example/lib/meeting/meeting_page.dart

```diff
@@ -713,6 +724,11 @@ class _MeetingPageState extends State<MeetingPage> with WidgetsBindingObserver {
   void didChangeAppLifecycleState(AppLifecycleState state) {
     super.didChangeAppLifecycleState(state);
     if (state == AppLifecycleState.resumed) {
+      _meetingStore.audioTracks.forEach((element) {
```

Comment:
the way we have to do set volume is in AppLifecycleState.resumed > call hmsSDK's getPeers function > then iterate over all peer.audioTrack & peer.auxiliaryTrack > then set volume: 10.0
white_check_mark
eyes
raised_hands
don't add another audioTracks array in meetingStore